packages/ActionBar/test/cypress/ActionBar.cypress.js

```diff
@@ -0,0 +1,201 @@
+const comp = "action-bar";
+const story = "showcase";
+
+describe("ActionBar", () => {
+  beforeEach(() => {
+    cy.visitStorybook(`${comp}--${story}`);

```

Comment:
this is necessary here? maybe just leave it as `cy.visitStorybook(`action-bar--showcase`);` or you can move this into the global visitStorybook helper

Reply:
Not really, I'll just leave it as cy.visitStorybook("action-bar--showcase");